src/components/core/common/LoadingScreen/style.ts

```diff
@@ -18,32 +18,6 @@ export const wrapper = styled.div`
   color: white;
   font-size: 30px;
 
-  .loader {
-    width: 32px;
-    height: 32px;
-    position: relative;
-    border-radius: 50%;
-    color: ${(props) => props?.theme?.colors?.primary};
-    animation: fill 1s ease-in infinite alternate;
-  }
-  .loader::before,
-  .loader::after {
-    content: "";
-    position: absolute;
-    height: 100%;
-    width: 100%;
-    border-radius: 50%;
-    left: 48px;
-    top: 0;
-    animation: fill 0.9s ease-in infinite alternate;
-  }
-
-  .loader::after {
-    left: auto;
-    right: 48px;
-    animation-duration: 1.1s;
-  }
-
```

Comment:
sao xoá hết vậy

Reply:
Load nớ bị to quá nên đổi lại cái dễ nhìn hơn =))